src/read.jl

```diff
@@ -71,26 +58,24 @@ function read_case(filepath)
     return case
 end
 
-"""Read input matfile (if present), return parsed data in a Storage struct."""
+"""Read input file (if present), return parsed data in a Storage struct."""
 function read_storage(filepath)::Storage
-    # Fallback dataframe, in case there's no case_storage.mat file
+    # Fallback dataframe, in case there's no input files
     storage = Dict(
         "enabled" => false, "gen" => zeros(0, 21), "sd_table" => DataFrames.DataFrame()
     )
     try
-        case_storage_file = MAT.matopen(joinpath(filepath, "case_storage.mat"))
-        storage_mat_data = read(case_storage_file, "storage")
-        println("...loading case_storage.mat")
+        println("...loading storage")
+        gen = DataFrames.DataFrame(CSV.File(joinpath(filepath, "storage_gen.csv")))
+
         # Convert N x 1 array of strings into 1D array of Symbols (length N)
-        column_symbols = Symbol.(vec(storage_mat_data["sd_table"]["colnames"]))
-        data = convert(Array{Float64,2}, storage_mat_data["sd_table"]["data"])
+        sd_path = joinpath(filepath, "StorageData.csv")
```

Comment:
Do we need to declare a new variable here?

Reply:
Nope - I was using it twice in a previous version but now it can be one line